### PR TITLE
Small tweaks to deployment documentation

### PR DESCRIPTION
**Pull request by timholy**
Despite the excellent documentation and clear statements about potential gotchas, I still missed a few crucial points. I don't know if this helps or hurts, but it's a couple of changes that address things that tripped me up.

### CODE REVIEW

File: docs/src/man/hosting.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ package repository:
 
 - travis buildbots startup and run your tests;
 - each buildbot will build the package docs using your `docs/make.jl` script;
-- a single buildbot will then try to push the generated docs back the github.
+- a single buildbot will then try to push the generated docs back to github.
+
+Note that the hosted documentation does not update when you make pull
+requests; you see updates only when you merge to `master`.
```
Comment (MichaelHatherly) [inline docs/src/man/hosting.md]: Also when new tags are pushed to the repo, as well as merging to `master`.

**Comment by MichaelHatherly** [comment]
Thanks! If you're able to narrow down which version of `travis` worked and didn't work that would be great to add as well.

**Comment by timholy** [comment]
Thanks for the suggestions!

**Comment by MichaelHatherly** [comment]
Great, thanks!

(Feel free to add any packages you document to the lists in https://juliadocs.github.io/Documenter.jl/latest/man/examples/ if you'd like.)

**Pull request merged**